pkg/console/subresource/deployment/deployment.go

```diff
@@ -332,3 +309,46 @@ func IsAvailableAndUpdated(deployment *appsv1.Deployment) bool {
 		deployment.Status.ObservedGeneration >= deployment.Generation &&
 		deployment.Status.UpdatedReplicas == deployment.Status.Replicas
 }
+
+func defaultVolumeConfig() []volumeConfig {
+	return []volumeConfig{
+		{
+			name:     ConsoleServingCertName,
+			readOnly: true,
+			path:     "/var/serving-cert",
+			isSecret: true,
+		},
+		{
+			name:     ConsoleOauthConfigName,
+			readOnly: true,
+			path:     "/var/oauth-config",
+			isSecret: true,
+		},
+		{
+			name:        api.OpenShiftConsoleConfigMapName,
+			readOnly:    true,
+			path:        "/var/console-config",
+			isConfigMap: true,
+		},
+		{
+			name:        api.ServiceCAConfigMapName,
+			readOnly:    true,
+			path:        "/var/service-ca",
+			isConfigMap: true,
+		},
+	}
+}
+
+func hasCustomLogo(operatorConfig *operatorv1.Console) bool {
+	if logoName := operatorConfig.Spec.Customization.CustomLogoFile.Name; logoName != "" {
+		return true
+	}
+	return false
+}
+
+func customLogoVolume(operatorConfig *operatorv1.Console) volumeConfig {
```

Comment:
not sure why we need the `operatorConfig *operatorv1.Console` ?

Reply:
maybe ?
```suggestion
func defaultCustomLogoVolume(operatorConfig *operatorv1.Console) volumeConfig {
```

Reply:
Will remove the arg.